test/test_client.py

```diff
@@ -202,8 +202,8 @@ def test_get_cpu_state(self):
         self.client.get_cpu_state()
 
     def test_set_session_password(self):
-        password = 'abcdefgh'
-        self.client.set_session_password(password)
+        pw = 'abcdefgh'
```

Comment:
also this is no improvement, i think if we would run pylint it would even say it is a bad variable.

Reply:
This was claimed by Codefactor, because of a "hardcoded password".